delivery-slots.js

```diff
@@ -1,21 +1,24 @@
-const puppeteer = require("puppeteer");
+const fs = require("fs");
 const ini = require("ini");
-var fs = require("fs");
-var shell = require("shelljs");
-var Push = require("pushover-notifications");
+const schedule = require("node-schedule");
+const puppeteer = require("puppeteer");
+const Push = require("pushover-notifications");
+const shell = require("shelljs");
 
 // Read config
 const config = ini.parse(fs.readFileSync("./config.ini", "utf-8"));
 
-// Log time
-var executiontime = Date.now();
-var date = new Date(executiontime);
-console.log(executiontime + ' ' + date.toUTCString());
+function getBrowser() {
```

Comment:
What's the advantage of doing compared to the existing way?

Reply:
Its awkward installing all the x11 dependencies for chromium to work in the node docker container. This allows instead a specifically suited docker image that has everything needed preinstalled. It does mean that memory for the browser is used for longer, but shouldn't be much of a problem with no pages open